CMakeLists.txt

```diff
@@ -129,7 +138,10 @@ if(${TRITON_ENABLE_NVTX})
 endif() # TRITON_ENABLE_NVTX
 
 find_package(ZLIB REQUIRED)
-find_package(Threads REQUIRED)
+
+if(NOT WIN32)
+  find_package(Threads REQUIRED)
```

Comment:
I wonder why `Threads` is not available on Windows?

Reply:
I tested locally including Threads and there was no issue. This is probably an early stage removal that was never revisited. I'll run a pipeline to verify functionality.